pilot/pkg/serviceregistry/aggregate/controller.go

```diff
@@ -36,16 +38,26 @@ var (
 var _ model.ServiceDiscovery = &Controller{}
 var _ model.Controller = &Controller{}
 
+// MeshConfigFn returns the MeshConfig.
+type MeshConfigFn func() *meshconfig.MeshConfig
```

Comment:
If we don't already have a pattern to access MeshConfig - this should be added to meshconfig package, not in serviceregistry.

Reply:
done. reused existing mesh.Holder